### PR TITLE
Delegate toString to wrapped class

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
Delegate toString to wrapped class

### What issues does this PR fix or reference?
n/a
#### Release Notes
n/a
#### Docs PR
n/a

### CODE REVIEW

**Comment by skabashnyuk** [comment]
ci-test

**Review by sleshchenko — APPROVED** [review]
LGTM 
Please take a look my inlined comment

**Comment by skabashnyuk** [comment]
ci-test

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1195/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1195//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11959
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

File: core/commons/che-core-commons-lang/src/main/java/org/eclipse/che/commons/lang/concurrent/CopyThreadLocalRunnable.java [hunk 1]
```diff
@@ -42,6 +42,11 @@ public void run() {
     }
   }
 
+  @Override
+  public String toString() {
+    return "CopyThreadLocalRunnable{ " + wrapped.toString() + " )";
```
Comment (riuvshin) [inline core/commons/che-core-commons-lang/src/main/java/org/eclipse/che/commons/lang/concurrent/CopyThreadLocalRunnable.java]: typo?

**Comment by skabashnyuk** [comment]
ci-test

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1196/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1196//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11959
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Pull request merged**